SoftLayer/managers/network.py

```diff
@@ -685,3 +692,38 @@ def get_nas_credentials(self, identifier, **kwargs):
         """
         result = self.network_storage.getObject(id=identifier, **kwargs)
         return result
+
+    def edit(self, instance_id, name=None, note=None, tags=None):
+        """Edit a vlan.
+
+        :param integer instance_id: the instance ID to edit.
+        :param string name: valid name.
+        :param string note: note about this particular v;am.
```

Comment:
should be: `:param string note: note about this particular vlan.`

Reply:
Done.